src/react-components/src/__tests__/components/HelperErrorText-test.tsx

```diff
@@ -1,15 +1,21 @@
-
 import { expect } from "chai";
 import * as Enzyme from "enzyme";
 import * as React from "react";
 import * as Mocha from "mocha";
 
-import HelperErrorText from "../../components/02-molecules/Helpers/HelperErrorText";
+import HelperErrorText from "../../components/01-atoms/Forms/HelperErrorText";
 
 describe("HelperErrorText Test", () => {
-  it("Throws error when HElperErrorText is called with no content", () => {
-    expect(() => Enzyme.mount(<HelperErrorText id="helperTextWithLink"
-    isError={false}/>))
-    .to.throw("Helper and Error Text must have content, or else it should not be used");
+  it("Renders HelperErrorText", () => {
+    const container = Enzyme.mount(<HelperErrorText id="helperTextWithLink"
+    isError={false}>Text</HelperErrorText>);
+    expect(container.exists("#helperTextWithLink")).to.equal(true);
+  });
+
+  it("Has 'error' modifier if error is passed", () => {
+    const container = Enzyme.mount(<HelperErrorText id="helperTextWithLink"
+    isError={true}>Text</HelperErrorText>);
+    console.log("debug", container.debug());
```

Comment:
debug statement.

Reply:
Actually, this test isn't done yet, right? It's not really testing anything that the above test isn't.